lib/GoalCohort.js

```diff
@@ -114,6 +115,18 @@ GoalCohort.statics.findUserCohort = async function(user, goal) {
   });
 };
 
+/**
+ * Finds the GoalCohort with the given name
+ */
+GoalCohort.statics.findCohortByName = async function(name, goal) {
```

Comment:
I think we should also have findCohortByName here work regardless of case, etc. So similar to previous quests we should have tests that ensure that `findCohortByName` works with `chiefs` or `ChiEfs`